core/src/view/view.cpp

```diff
@@ -134,7 +135,12 @@ void View::setRoll(float _roll) {
 
 void View::setPitch(float _pitch) {
 
-    m_pitch = glm::clamp(_pitch, 0.f, (float)HALF_PI);
+    float max_pitch = HALF_PI;
+    if (m_type != CameraType::perspective) {
+        // Prevent projection plane from intersecting ground plane
+        max_pitch = atan2(m_pos.z, m_height * .5f);
```

Comment:
Oh yeah, I hadn't tried changing the orientation - that's an interesting problem. `m_height` is still the value to check against here, but its value changes when the screen is re-oriented and the pitch won't be checked for validity until the next time its set. A quick fix here might be to call `setPitch(m_pitch)` in the `resize` call - I'll try that.